### ISSUE TITLE
Accessibility: Usability: Narrator: Narrator announces wrong item number available under "In this article" header

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#6]
#A11yUsable
#Wipro
#vscode-website
#Firefox

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Version-10.0.14393 Build 14393.607
Browser-Edge
Edge Version-Microsoft Edge 38.14393.607.0

Firefox version:
50.1.0

**Repro Steps:**
1. Open https://code.visualstudio.com/docs/introvideos/overview in EDGE
2. Navigate to the "In this article" Header in the Navigation Landmark in the RHS of the page (with Caps Lock + --> keys)
3. Then again press Caps Lock + --> keys
4. Observe the Narrator announcement and the focus

**Actual Result:**
The Narrator focus goes to some other place in the browser and announces as "Contains 0 items" though the Tweet link is present under the "In this article" header.

**Inspect Behavior:**
Refer Inspect Screenshot**Expected Result:**
The Narrator should announce the name of the symbol.

**Expected Result:**
The narrator focus should be on the correct item and should announce the correct list item available under the "In this article" header.

**User Impact:**
Users who rely on screen readers will be  updated with wrong data and they will be misguided by these type of wrong information

![narrator announcement screenshot](https://cloud.githubusercontent.com/assets/24825441/21673910/56c719de-d351-11e6-9056-c8110e2458d9.png)
![narrator buddy screenshot](https://cloud.githubusercontent.com/assets/24825441/21673915/5b958798-d351-11e6-96ca-8d6a53cd8553.png)
![inspect behaviour screenshot](https://cloud.githubusercontent.com/assets/24825441/21674027/f7aad818-d351-11e6-9043-462775f96104.png)

**Comment by waderyan** [comment]
Fixed
